docs/source/user_guide.rst

```diff
@@ -190,6 +190,40 @@ example, with pandas.
    >>> url = export_result.output[0].path
 
 
+Logging
+=======
```

Comment:
This is what this new section would look like in the compiled Sphinx docs:

<img width="729" alt="Screenshot 2024-05-10 at 8 42 04 AM" src="https://github.com/civisanalytics/civis-python/assets/86482098/9a6ebae0-e357-4366-a071-c721b0b9253e">